### PR TITLE
Admin asset list

### PR DESCRIPTION
**Pull request by davidgljay**
## What does this PR do?

Creates a list where admins can review assets with comment streams.

## How do I test this PR?

1. Create a few streams by visiting `http://localhost:300/assets/some-random-string`
2. Visit the admin panel.
3. Click "Streams" tab.
4. You should see a list of streams.
5. Sort, filter and search should work as expected.
6. Pagination should work as expected if more than 25 streams are present.

## Note
There is an unrelated bug causing publication dates to be incorrect.

### CODE REVIEW

**Comment by riley** [comment]
If I close an article in the toggle thing, it doesn't flip over to show "closed" until I reload the page.

**Comment by riley** [comment]
if I click Sort By any of the filters, the list disappears.

**Comment by riley** [comment]
typing in the search box makes the articles disappear also

**Comment by riley** [comment]
clicking Next on the pagination thing at the bottom takes me to the next page (I think?) and I can't get back to see the articles. I'd probably just disable Next if there is no next page

**Comment by davidgljay** [comment]
Bug fixed and `Next` and `Prev` buttons removed.

File: client/coral-admin/src/containers/Streams/Streams.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+      prevState.timer = setTimeout(() => {
+        fetchAssets(0, limit, this.state.search, this.state.sort, this.state.filter);
+      }, 350);
+      return prevState;
+    });
+  }
+
+  renderDate = (date) => {
+    const d = new Date(date);
+    return `${d.getMonth() + 1}/${d.getDate()}/${d.getFullYear()}`;
```
Comment (riley) [inline client/coral-admin/src/containers/Streams/Streams.js:54]: is there any way to detect a non-english language and display the dates in non-american style?

File: client/coral-admin/src/containers/Streams/Streams.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  onSettingChange = (setting) => (e) => {
+    let options = this.state;
+    this.setState({[setting]: e.target.value});
+    options[setting] = e.target.value;
+    this.props.fetchAssets(0, limit, options.search, options.sort, options.filter);
+  }
+
+  onSearchChange = (e) => {
+    this.setState({search: e.target.value});
+    this.setState((prevState) => {
```
Comment (riley) [inline client/coral-admin/src/containers/Streams/Streams.js:41]: is there anyway to combine these two `this.setState` statements?

File: client/coral-admin/src/containers/Streams/Streams.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    this.setState({page});
+    const {search, sort, filter} = this.state;
+    this.props.fetchAssets((page - 1) * limit, limit, search, sort, filter);
+  }
+
+  render () {
+    const {search, sort, filter} = this.state;
+    const {assets} = this.props;
+
+    console.log(assets);
```
Comment (riley) [inline client/coral-admin/src/containers/Streams/Streams.js]: remove this log

**Review by riley — CHANGES_REQUESTED** [review]
looking great, everything works. just a couple of optimizations if possible

File: client/coral-admin/src/containers/Streams/Streams.js (hunk 1 above)
Comment (gabelula) [reply client/coral-admin/src/containers/Streams/Streams.js:54]: In that case date format needs to be saved as part of the Settings.

Comment (davidgljay) [reply client/coral-admin/src/containers/Streams/Streams.js:54]: I thought about this, but it seemed out of scope for this story. It involves an unrelated setting and associated UI.

**Comment by davidgljay** [comment]
Recommended updates implemented.

**Review by riley — APPROVED** [review]
awesome! 💯

**Pull request merged**